### PR TITLE
switch to radio buttons for filtering

### PR DESCRIPTION
**Pull request by mashehu**
Following the good idea mentioned in https://github.com/nf-core/nf-co.re/pull/1423?notification_referrer_id=MDE4Ok5vdGlmaWNhdGlvblRocmVhZDQ3NDAyNDA1ODE6NjE2OTAyMQ%3D%3D#issuecomment-1300793764

<a href="https://gitpod.io/#https://github.com/nf-core/nf-co.re/pull/1424"></a>

### CODE REVIEW

**Comment by mashehu** [comment]
@nf-core-bot fix linting

File: public_html/events.php [hunk 1]
```diff
@@ -8,7 +8,7 @@
 $event_type_classes = [
     'bytesize' => 'success',
     'hackathon' => 'primary',
-    'poster' => 'secondary',
+    'poster' => 'danger',
     'talk' => 'success',
     'tutorial' => 'info',
     'workshop' => 'warning',
```
Comment (ewels) [inline public_html/events.php:14]: Should we rename this to "training"?

**Review by ewels — APPROVED** [review]
Code LGTM 👍🏻

File: public_html/events.php (hunk 1 above)
Comment (ewels) [reply public_html/events.php]: Done

**Pull request merged**